app/Http/Controllers/ProjectController.php

```diff
@@ -16,31 +16,31 @@ class ProjectController extends Controller
      */
     public function index(Request $request)
     {
-        if ($request->get('pid')) {
-            $projects = Project::find(16);
-            $projects->members->pluck('id')->toArray();
+      if ($request->get('pid')) {
+        $projects = Project::find(16);
+        $projects->members->pluck('id')->toArray();
+      }
+      $projects = Project::all();
```

Comment:
You probably want to use a paginator here instead of returning all projects:
`https://laravel.com/docs/5.5/pagination`